### PR TITLE
docs: add the realm push commands

### PR DESCRIPTION
**Pull request by lornajane**
## What/Why/How?

We wrap the `push` and `push-status` commands, so bring those commands into our docs.
Add tabs to the `push` page so users can pick which version they're using.

## Check yourself

- [ ] Code changed? - Tested with redoc/reference-docs/workflows (internal)
- [ ] All new/updated code is covered with tests
- [ ] New package installed? - Tested in different environments (browser/node)

## Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

**Comment by lornajane** [comment]
Not really sure what to do about the repeated headings errors - since we do have the same structure, one per product.

File: docs/commands/push-status.md
Comment (JLekawa) [inline docs/commands/push-status.md:6]:
```suggestion
The `push-status` command can be used whenever the application or process executing a `push` command (without `--wait-for-deployment` option) returns the `pushId`.
```

File: docs/commands/push-status.md
Comment (HCloward) [inline docs/commands/push-status.md:3]: Should this be an admonition? Make it stand out more?

File: docs/commands/push-status.md
Comment (HCloward) [inline docs/commands/push-status.md:41]: There are some issues with the partial that I am going to address in a corresponding PR. I will also look into making the section linkable so we don't need to include a partial.

Comment (HCloward) [reply docs/commands/push-status.md:41]: The corresponding PR is https://github.com/Redocly/redocly/pull/10002

File: docs/commands/push-status.md
Comment (lornajane) [reply docs/commands/push-status.md:3]: I try to use fewer markdoc extras in the CLI repo since anyone reading the docs on GitHub won't see that content rendered well. And I'm not sure how many people will try to use this command and find it confusing .... so I erred on the side of leaving it as words for now. We might well need to revisit later!

File: docs/commands/push-status.md
Comment (lornajane) [reply docs/commands/push-status.md]: Thanks!

**Comment by lornajane** [comment]
:warning:  Do not merge. The build looks healthy but the push page is actually broken in the preview (and locally). This must be corrected before merging. https://marketing-site--rem-git-rem-01hnb1dt4c5k4ncv0g3k-dc9cc3.preview.redocly.app/docs/cli/commands/push

**Comment by lornajane** [comment]
Note: add a redirect from the old reference pages inside beta docs after merging this PR

File: docs/commands/push.md
Comment (HCloward) [inline docs/commands/push.md:377]: I wasn't real sure how this "default branch" differs from the "branch" option.

Comment (lornajane) [reply docs/commands/push.md:377]: It's the branch on the origin repository, rather than the one we're publishing to

Comment (lornajane) [reply docs/commands/push.md]: I can't figure out how to say that in different words that isn't more confusing ...

File: docs/commands/push.md
Comment (adamaltman) [inline docs/commands/push.md:8]: I think we should probably switch the order and put Reunite before Workflows.

**Comment by adamaltman** [comment]
I think we need to merge this because we're going to launch the updated website.

**Review by adamaltman — APPROVED** [review]

File: docs/commands/push.md
Comment (RomanHotsiy) [reply docs/commands/push.md:8]: 👍

**Review by RomanHotsiy — APPROVED** [review]

**Pull request merged**
